### PR TITLE
fix: removed validation from variables

### PR DESCRIPTION
**Pull request by Aayush-Abhyarthi**
### Description

Replace this text with a summary of the changes in this PR. Include why the changes are needed and context about the changes. List required dependencies. If there is a Git issue for the change, please link to it.

#### No release required

- [ ] Examples or tests (addition or updates of examples or tests)
- [ ] Documentation update
- [ ] CI-related update (pipeline, etc.)
- [ ] Other changes that don't affect Terraform code

#### Release required

- [ ] Bug fix (patch release (`x.x.X`): Change that fixes an issue and is compatible with earlier versions)
- [ ] New feature (minor release (`x.X.x`): Change that adds functionality and is compatible with earlier versions)
- [ ] Breaking change (major release (`X.x.x`): Change that is likely incompatible with previous versions)

##### Release notes content

Replace this text with information that users need to know about the bug fixes, features, and breaking changes. This information helps the merger write the commit message that is published in the release notes for the module.

---

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### Merge actions for mergers

- Merge by using "Squash and merge".
- Use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents and any release notes provided by the PR author.

    The commit message determines whether a new version of the module is needed, and if so, which semver increment to use (major, minor, or patch).

### CODE REVIEW

**Comment by ocofaigh** [comment]
@Aayush-Abhyarthi please resolve conflicts so PR can be merged

File: variables.tf [hunk 1]
```diff
@@ -109,44 +109,20 @@ variable "backup_encryption_key_crn" {
...
-      var.disk_mb <= 4194304
-    ])
-    error_message = "Member group disk must be >= 5120 and <= 4194304 in increments of 1024."
-  }
 }
 
 variable "cpu_count" {
   type        = number
-  description = "Number of CPU cores available to the mongodb instance"
+  description = "The number of CPU cores available to the database instance. For more information refer to the docs https://cloud.ibm.com/docs/databases-for-mongodb?topic=databases-for-mongodb-pricing"
```
Comment (ocofaigh) [inline variables.tf:124]: This is pointing to pricing section of that doc. You prob want to point to https://cloud.ibm.com/docs/databases-for-mongodb?topic=databases-for-mongodb-pricing#mongodb-scale-member

File: variables.tf [hunk 2]
```diff
@@ -109,44 +109,20 @@ variable "backup_encryption_key_crn" {
...
-      var.memory_mb <= 114688
-    ])
-    error_message = "Member group memory must be >= 1024 and <= 114688 in increments of 128."
-  }
 }
 
 variable "disk_mb" {
   type        = number
-  description = "Disk available to the mongodb instance"
+  description = "Disk space available to the database instance. For more information refer to the docs https://cloud.ibm.com/docs/databases-for-mongodb?topic=databases-for-mongodb-pricing"
```
Comment (ocofaigh) [inline variables.tf:118]: same as https://github.com/terraform-ibm-modules/terraform-ibm-icd-mongodb/pull/151/files#r1182952157

File: variables.tf [hunk 3]
```diff
@@ -109,44 +109,20 @@ variable "backup_encryption_key_crn" {
 
 variable "memory_mb" {
   type        = number
-  description = "Memory available to the mongodb instance"
+  description = "Memory available to the database instance in MB. For more information refer to the docs https://cloud.ibm.com/docs/databases-for-mongodb?topic=databases-for-mongodb-pricing"
```
Comment (ocofaigh) [inline variables.tf:112]: same as https://github.com/terraform-ibm-modules/terraform-ibm-icd-mongodb/pull/151/files#r1182952157

**Review by ocofaigh — CHANGES_REQUESTED** [review]
see comments

File: variables.tf (hunk 1 above)
Comment (Aayush-Abhyarthi) [reply variables.tf]: Updated

File: variables.tf (hunk 3 above)
Comment (Aayush-Abhyarthi) [reply variables.tf]: Updated

File: variables.tf (hunk 2 above)
Comment (Aayush-Abhyarthi) [reply variables.tf]: Updated

**Review by ocofaigh — APPROVED** [review]

**Pull request merged**